pkg/manager/client/k8s.go

```diff
@@ -51,6 +51,23 @@ func (k *KubernetesClient) GetPodContainerLogRequest(namespace, podName, contain
 	})
 }
 
+func (k *KubernetesClient) GetPodContainerPreviousLogRequest(namespace, podName, containerName string) *rest.Request {
+	return k.clientSet.CoreV1().Pods(namespace).GetLogs(podName, &corev1.PodLogOptions{
+		Container:  containerName,
+		Timestamps: true,
+		Previous:   true,
+	})
+}
+
+func (k *KubernetesClient) GetPodRestartCount(namespace, podName, containerName string) int32 {
+	pod, _ := k.clientSet.CoreV1().Pods(namespace).Get(context.TODO(), podName, metav1.GetOptions{})
```

Comment:
Should handle the error.

Reply:
Ah, good catch! Thanks for the reminder.
Fixed!